src/platform/wiiu/main.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <cstdlib>
+#include <unistd.h>
+#include <string>
+#include <vector>
+#include <chrono>
+#include "player.h"
+#include "utils.h"
+#include "output.h"
+
+#ifdef USE_SDL // SDL might wrap main()
+#  include <SDL.h>
+#endif
+
+#include <coreinit/debug.h>
+#include <coreinit/foreground.h>
+#include <proc_ui/procui.h>
+#include <sysapp/launch.h>
+
+using namespace std::chrono_literals;
+
+#ifdef NDEBUG
+// stubbed
+static void initLogging() {}
+static void deinitLogging() {}
+#else
+#include <whb/log.h>
+#include <whb/log_cafe.h>
+#include <whb/log_module.h>
+#include <whb/log_udp.h>
+
+static bool moduleLogInit = false;
+static bool cafeLogInit = false;
+static bool udpLogInit = false;
+static bool prodUiExited = false;
```

Comment:
This needs to be moved above the `#ifdef NDEBUG` in line 38 and should be changed to `procUiExited`.